devel/n/Portfile

```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
-github.setup        tj n 9.2.0 v
+github.setup        tj n 9.2.1 v
 revision            1
```

Comment:
`revision` needs to be reset to zero when `version` gets bumped.

Reply:
fixed. IDK how I always forget that...